tethysapp/water_data_explorer/endpoints.py

```diff
@@ -18,42 +18,36 @@
 logging.getLogger('suds.client').setLevel(logging.DEBUG)
 
 
-@controller(
-    name='get-download-hs',
-    url='get-download-hs/',
-)
+@controller(name='get-download-hs', url='get-download-hs/')
 def get_download_hs(request):
     hs_name = request.POST.get('hs_name')
     hs_url = request.POST.get('hs_url')
     variable_hs = request.POST.get('variable_hs')
     site_hs = request.POST.get('site_hs')
-    url = ('https://gist.githubusercontent.com/romer8/89c851014afb276b0f20cb61c9c731f6/raw/a0ee55ca83e75f34f26eb94bd52941cc2a2199cd/pywaterml_template.ipynb')
+    url = ('https://gist.githubusercontent.com/romer8/89c851014afb276b0f20cb61c9c731f6/raw/\
+           a0ee55ca83e75f34f26eb94bd52941cc2a2199cd/pywaterml_template.ipynb')
```

Comment:
This is the more common way to break long strings:

```suggestion
    url = ('https://gist.githubusercontent.com/romer8/89c851014afb276b0f20cb61c9c731f6/raw/'
              'a0ee55ca83e75f34f26eb94bd52941cc2a2199cd/pywaterml_template.ipynb')
```